src/main/sensors/gyroanalyse.h

```diff
@@ -53,10 +58,11 @@ typedef struct gyroAnalyseState_s {
 
     biquadFilter_t detectedFrequencyFilter[XYZ_AXIS_COUNT];
     uint16_t centerFreq[XYZ_AXIS_COUNT];
+    uint16_t prevCenterFreq[XYZ_AXIS_COUNT];
```

Comment:
previous center frequency is already a member of `detectedFrequencyFilter`

Reply:
Can't figure out what to do to fix this, sorry

Reply:
If I comment out 61, it won't compile... `c:135:16: error: 'gyroAnalyseState_t {aka struct gyroAnalyseState_s}' has no member named 'prevCenterFreq';....`

Reply:
`biquadFilter_t::y1` contains the output of filter from previous sample if you're using `biquadFilterApplyDF1`.
I didn't notice we're using just the `biquadFilterApply`.